src/next.md

```diff
@@ -7,6 +7,189 @@
 This section describes what might happen after a changeset gets into the build,
 for example a bug might be filed or a backport could be requested.
 
+* [Requesting a Backport](#requesting-a-backport)
+* [Filing a Bug](#filing-a-bug)
+* [ProblemListing or `@ignore`-ing a Test](#problemlisting-or-ignore-ing-a-test)
+* [Backing Out a Change](#backing-out-a-change)
+
+## Requesting a Backport
+
+TBD
+
+## Filing a Bug
+
+When a new failure is found in the JDK a bug should be filed to describe and track the issue. Depending on your role in the OpenJDK you can either use the [Bug Report Tool](https://bugreport.java.com/) or, if you are Author in an OpenJDK Project, report the bug directly in the [JDK Bug System](https://bugs.openjdk.java.net/). Try to make the bug report as complete as possible to make it easier to triage and investigate the bug. It's not the reporter's responsibility to set a correct priority, but a qualified guess is always better than a default value. To help with setting the right priority consider things like how the bug impacts the product and our testing, how likely is it that the bug triggers, how difficult is it to work around the bug if it's not fixed soon, and whether it's a recent regression, since that may break existing applications. Regressions are often higher priority than long standing bugs and may block a release if not fixed.
+
+A few things to keep in mind when filing a new bug:
+
+* Before filing a bug, verify that there isn't already a bug filed for this issue.
+* Add relevant labels like `intermittent`, `regression`, `noreg-self` etc.
+* Set affects version to the JDK version(s) where the failure was seen.
+  * If the failure is found in an update train of the JDK (e.g. 11.0.x), please make an effort to see if the bug is also present in [mainline](https://hg.openjdk.java.net/jdk/jdk/).
```

Comment:
This isn't really a comment on your proposed change. I presume there will be a follow-on task to update the docs after next week when the jdk repo transitions from Mercurial to Git?

Reply:
Yes, there is a Skara bug filed to update the guide, https://bugs.openjdk.java.net/browse/SKARA-652